src/components/Table/Table/Table.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, ReactElement } from "react";
+import cx from "classnames";
+import { VibeComponentProps } from "../../../types";
+import styles from "./Table.module.scss";
+import { ITableHeaderProps } from "../TableHeader/TableHeader";
+import { ITableBodyProps } from "../TableBody/TableBody";
+import { getTableRowLayoutStyles } from "./tableHelpers";
+
+export interface ITableColumn {
+  id: string;
+  title: string;
+  infoContent?: string;
+  width?: number | { min: number; max: number };
+}
+
+interface ITableProps extends VibeComponentProps {
+  columns: ITableColumn[];
+  dataState: {
```

Comment:
It's not really clear what the purpose of this prop... can you explain?

Reply:
is these are booleans for errorState and emptyState?

Reply:
You're right :) Since we don't support `onRequestMoreData ` yet, then `hasMoreResults` is out of context, and I'll remove it.

`isError` is indeed a boolean for the error state